client/src/styles/index.js

```diff
@@ -0,0 +1,36 @@
+import styled from "styled-components";
+import { NavLink } from 'react-router-dom';
+
+
+export const NavComp = styled.nav`
+    display: flex;
+    background: #335C67;
+    padding: 15px 0;
+    justify-content: flex-end;
+    margin-bottom: 30px;
+    img {
+        width: 20px;
+        height: 20px;
+        margin-right: 3%;
+        &:hover {
+            cursor: pointer;
+        }
+    }
+`;
+
+const activeClassName = 'active-nav-item';
+
+export const NavItem = styled(NavLink).attrs({
```

Comment:
these styled components make me happy. Great job going the extra mile and adding in the react portion of your app.